flytekit/models/launch_plan.py

```diff
@@ -37,14 +40,24 @@ def notifications(self):
         """
         return self._notifications
 
+    @property
+    def launch_conditions(self):
+        return self._launch_conditions
+
     def to_flyte_idl(self):
         """
         List of notifications based on Execution status transitions
         :rtype: flyteidl.admin.launch_plan_pb2.LaunchPlanMetadata
         """
+        if self.launch_conditions:
+            a = Any()
```

Comment:
why is [launch_conditions](https://github.com/flyteorg/flyte/blob/5e00e7b03b308d1ce6cd7b3bffa2589e568c04b5/flyteidl/protos/flyteidl/admin/launch_plan.proto#L170) of type protobuf.Any? Shouldn't it be a struct?